kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js

```diff
@@ -38,10 +38,12 @@ define(['bluebird', 'common/html', './jobStateList'], (Promise, html, JobStateLi
         }
 
         function stop() {
-            container.innerHTML = '';
-            if (jobStateListWidget) {
-                jobStateListWidget.stop();
-            }
+            return Promise.try(() => {
```

Comment:
This promise was required after all -- you can't toggle to another tab without a resolved promise from the active tab.